packages/dids/package.json

```diff
@@ -77,9 +77,12 @@
   "dependencies": {
     "@decentralized-identity/ion-sdk": "1.0.1",
     "@dnsquery/dns-packet": "6.1.1",
+    "@types/ms": "0.7.31",
```

Comment:
Shouldn't `@types/ms` be under `devDependencies `?

Reply:
Yes!

Reply:
Addressed in [4442df4](https://github.com/TBD54566975/web5-js/pull/419/commits/4442df4f47b676650cbb74fa85b0c48917fc600f)